index.html

```diff
@@ -0,0 +1,236 @@
+                </div>
+              </div>
+            </div>
+           
+            <div class="col-md col-12 mb-4 no_margin">
+              <div class="card text-center h-100 card-color">
+                <div class="card-body card_layout">
+                  <span class="material-symbols-outlined text-white">
+                    workspaces
+                    </span>
+                  <h5 class="card-title pt-3">Workshop</h5>
+                  <p class="card-text pt-2 card_info">Try creating your own work using open source license rather than just watching at it.</p>
+                </div>
+              </div>
+            </div>
+           
+            <div class="col-md col-12 mb-4 no_margin">
+              <div class="card text-center h-100 card-color">
+                <div class="card-body card_layout">
+                  <span class="material-symbols-outlined text-white">
+                    settings
+                    </span>
+                  <h5 class="card-title pt-3">ES Ignite</h5>
+                  <p class="card-text pt-2 card_info">Get Opertunities to network with ES effiliates arount the world also after the Symposium.</p>
+                </div>
+              </div>
+            </div>
+           
+          </div>
+          <div class="row mt-5">
+            <div class="col-md-12 d-flex justify-content-center">
+              <a href="#" class="see_program text-white">SEE THE WHOLE PROGRAM</a>
+              <button class="see_more_btn btn btn-secondary">Join ES Global Summit 2023</button>
+            </div>
+
+          </div>
+        </div>
+        
+      </section>
+    
+      <section id="speakers" class="container-fluid  mt-5">
+        
+        
+      </section>
+      <section id="partners" class="container-fluid mt-5">
+        <div class="container text-white p-3">
+          <h3 id="partner_heading" class="text-center">Partners</h3>
+          <hr class="hr-partner">
+          
+        </div>
+        <div class="container pb-4 mt-2">
+          <div class="row">
+            <ul class="list-unstyled d-flex justify-content-center">
+              <li class="col my-auto">
+                <img src="images/mozilla 1.png" alt="" class="social_links w-100">
+              </li>
+              <li class="col my-auto">
+                <img src="images/googlekorea 1.png" alt="" class="social_links w-100">
+              </li>
+              <li class="col my-auto">
+                <img src="images/naver 1.png" alt="" class="social_links w-100">
+              </li>
+              <li class="col my-auto">
+                <img src="images/daumkakaologo 1.png" alt="" class="social_links w-100">
+              </li>
+              <li class="col my-auto">
+                <img src="images/airbnb 1.png" alt="" class="social_links w-100">
+              </li>
+            </ul>
+          </div>
+        </div>
```

Comment:
- [ ] (Optional) It would be nice if you could display the partner section as it has been done in the given design for the `index.html` mobile version.

Your Design
![image](https://github.com/kamranasif506/Educational-Symposium/assets/81932478/0acbe67b-7268-449b-bbe7-15135f38ccb1)

Given Design
![image](https://github.com/kamranasif506/Educational-Symposium/assets/81932478/b1cf44ec-3b7c-43a2-b77f-9eb801b1ee5e)